docs/operation/troubleshooting.rst

```diff
@@ -0,0 +1,54 @@
+
+.. _Troubleshooting Guide:
+
+Troubleshooting Guide
+^^^^^^^^^^^^^^^^^^^^^
+
+This section highlights some of the common problems users face during and
+after a MetalK8s installation. If you do not find a solution to a problem you
+are facing, please reach out to us or create a
+`Github issue <https://github.com/scality/metalk8s/issues>`_.
+
+Bootstrap Installation Errors
++++++++++++++++++++++++++++++
+
+Bootstrap Installation fails with no straightforward reason
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+If during a MetalK8s installation you encounter a failure and the console
+output does not provide sufficient information in order to pin-point the cause
+of failure, then re-run the installation with verbose arguments(--verbose).
+
+.. parsed-literal::
+
+   root@bootstrap $ /srv/scality/metalk8s-|release|/bootstrap.sh --verbose
+
+Errors after restarting the Bootstrap node
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+If you restart the Bootstrap node and for some reason, some containers
+(especially the salt-master container) refuses to start, check and ensure that
+the **MetalK8s ISO** is mounted properly.
+
+Bootstrap fails and console log is unscrollable
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+If during a MetalK8s installation, the Bootstrap process fails and the console
+output is unscrollable then you can consult the Bootstrap logs in
+``/var/log/metalk8s-bootstrap.log``.
+
+Account Administration Errors
++++++++++++++++++++++++++++++
+
+Forgot the MetalK8s GUI password
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+If you forget the MetalK8s GUI username and/or password combination,
+follow
+:ref:`this procedure <Administering-MetalK8s-GUI-Kubernetes-API-and-Salt-API>`
+to reset or change it.
+
+
+.. todo::
```

Comment:
are you going to handle this in this PR?

Reply:
I think the two last ones are not mission critical

Reply:
Troubleshooting for networking is indeed a large part and may be relegated to a future enhancement (worth opening a ticket to track this however, cc @thomasdanan).

For Salt logs, I think it would make sense to at least document it somewhere. Let's keep it for 2.4.2, but not this PR, which is already quite big.